### PR TITLE
Replaces native BigInt with a custom modulo function

### PR DESCRIPTION
**Pull request by MattivdWeem**
The BigInt implementation was not backwards compatible; i opted to use ~JSBI~ custom mod function.

We can't use `Number()` due to the way the number is generated we'll almost always get a number above 2^53. 

Open for better solutions, ~this was in my view the easiest with the least impact, but sadly requires a dependency.~ this was my second solution, doesn't require a dependency, but requires some custom math. 

fixes: #94

### CODE REVIEW

**Comment by dalawwa** [comment]
We are currently using v1.x only because of ever green support of v2.
Do you know if was the only change that introduced the difference in browser support ?

**Comment by belgac** [comment]
I also seems to me it's the only non compliant syntax with evergreen browsers.

I'm not sure relying directly on jsbi is the best solution though

I think we should build the typescript files with babel instead of ts and rely on the @babel/preset-env module and the polyfills so that the decision about wich polyfill to use when and how is not in the hands of this library alone anymore + anyone can rebuild with the correct browserlist for his environement

**Comment by belgac** [comment]
We could even trigger a double build to release a evergreen version as v2.x and to realease a backward compatible version as v1.x

**Comment by se-panfilov** [comment]
Well... That's sounds like a problem, honestly.

First of all, we should not force user to include `jsbi`. I mean - so far it makes sense only for `NL` and only for browsers. 

And I honestly believe that jsbi might be a `devDependency` at least (As I understand there is no difference while you are on the FE, but might be a problem if you use it on the server). But I'd prefer to get rid of that dependency at all.

Also I wonder, why `jsbi`, not polyfill?

I mostly like what @belgac said. But we should keep in mind that such polyfills might be already include or unwanted by the user. I mean - jsbi seems to be like 28k minified (if I'm correct), for instance.

My suggestion is following. Let's add a config step to the `jsvat`. if user wants to use NL in browsers and he wants to care about safari, he should do following:

```ts
import { jsvatConfig, checkVAT, netherlands } from 'jsvat'; // "jsvatConfig" is a new thing here
import JSBI from "jsbi"; //might be different lib or polyfill of course

jsvatConfig({bigIng: JSBI});
checkVAT('NL000000024B01', [netherlands])
```

For sure, this solution has some cons: we should implement a wrapper for BigInt calculation, which add complexity and also ask users to add one more dependency. And read the docs more carefully, huh.

I'd like to keep the starting process as simple as possible for the users, but...

**Comment by se-panfilov** [comment]
But before we will go any further, I'd like to ask:

Maybe we can prevent our calculations from going out of the Number range?
Idk, very stupid solution, like divide all the numbers by 2 at some step, or whatever?

My point is following: if we can solve this problem at the level of the math it will be the best

**Comment by MattivdWeem** [comment]
@se-panfilov @wilgert i've changed JSBI towards a new mod function

**Comment by se-panfilov** [comment]
Looks good, will test it a bit. I can see some lint issues, but nothing major

File: src/lib/countries/netherlands.ts [hunk 1]
```diff
@@ -56,5 +56,19 @@ function getCharValue(char: string): number {
...
+
+// custom module function, to check module on values above Number limit
+function mod(value: string, divisor: number): number {
+  // Initialize result
+  let res: number = 0;
+
+  // iterate over Value and calculate the mod for each of them
+  for (let i = 0; i < value.length; i++) {
+    res = (res * 10 + +value[i]) % divisor;
+  }
```
[the remark this answers is not in the document]
Comment (se-panfilov) [reply src/lib/countries/netherlands.ts:70]: @MattivdWeem thanks.

Btw, I don't think that it is any difference between `for (const char of value.split('')) {` and `for (const char of value) {`, but it's okay for now

**Pull request merged**